### ISSUE TITLE
[BUG] URL pattern screen is not being loaded

### ISSUE THREAD

**Issue by SalahAdDin** [pluginpal/strapi-webtools#150]
### Describe the bug

After setting up both the the sitemap plugins with the core, we want to generate the URL patterns, we go to the screen, and it does not load.

### Steps to reproduce the behavior

1. Go to Admin Panel
2. Click on Webtools
3. Click on URL Patterns

### Expected behavior
We should see the screen available so we can edit the URL patterns.

### Screenshots

![image](https://github.com/pluginpal/strapi-webtools/assets/5034215/a253b8fa-28a2-43fe-b0bd-73c2d075ed02)

### Code snippets

If applicable, add code samples to help explain your problem.

### System

- Node.js version: v20.12.2
- NPM version: 10.5.0
- Strapi version: 4.24.4
- Plugin version: 1.0.0-beta.11
- Database: SQLLite
- Operating system: Manjaro Linux

### Additional context

I was using an old version of Sitemaps and I defined the URLs also; as there is no migration guide, I didn't migrate anything.

**Comment by SalahAdDin** [comment]
@boazpoolman ?

**Comment by SalahAdDin** [comment]
@boazpoolman 

From my investigation, I get that [fetching the URL patterns](https://github.com/pluginpal/strapi-webtools/blob/d683cd5ab265433e2e81118ddd28629657c23270/packages/core/admin/screens/Patterns/ListPage/index.tsx#L28) always [returns](https://github.com/pluginpal/strapi-webtools/blob/d683cd5ab265433e2e81118ddd28629657c23270/packages/core/admin/screens/Patterns/ListPage/index.tsx#L32) an empty array:
![image](https://github.com/pluginpal/strapi-webtools/assets/5034215/26bcc53c-091c-43fa-868e-2df1cfcfd61c)

It is a problem since the `index` [requires to get not an empty](https://github.com/pluginpal/strapi-webtools/blob/d683cd5ab265433e2e81118ddd28629657c23270/packages/core/admin/screens/Patterns/ListPage/index.tsx#L45) [array of patterns](https://github.com/pluginpal/strapi-webtools/blob/d683cd5ab265433e2e81118ddd28629657c23270/packages/core/admin/screens/Patterns/ListPage/index.tsx#L21):
```
if (loading || !patterns.length) {
    return (
      <Center>
        <Loader>{formatMessage({ id: 'webtools.settings.loading', defaultMessage: 'Loading content...' })}</Loader>
      </Center>
    );
  }
```

Is it intended to be like that? Is there any extra step to add URL patterns?

From the documentation we get that screen is where we define the patterns!

**Comment by MSACC** [comment]
Hello @SalahAdDin , thanks for sending in the bug report. We want to create robust Strapi plug-ins and thus want to help you (and maybe others) with this experience.

Could you provide us with some more information? What are the contents of your `wt_url_patterns` table in the database? Maybe it is empty or missing a value?

Thanks for letting us know!

Mathijs

**Comment by TMSchipper** [comment]
Hi @SalahAdDin,  I have tried to reproduce this issue and I can confirm that this is a problem.
I'll update the label and we'll take a look at it soon. Hopefully I have informed you sufficiently, if you have any questions you can always contact us.

Kind regards,
PluginPal.

**Comment by boazpoolman** [comment]
This issue has been fixed and released with [v1.0.0-beta.12](https://github.com/pluginpal/strapi-webtools/releases/tag/%40pluginpal%2Fwebtools-core%401.0.0-beta.12)
